helpers/isCarParking.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = function(dust) {
+
+ /*
+  * @description Detect whether the upgrade type is car parking for conditional rendering in confirmation template
+  * @param upgradeName is the name of the upgrade type which is used to check if it is car parking
+  * @example {@_checkIfCarParking upgradeName=content.name /} returns true if Car Parking otherwise false
+  */
+
+  dust.helpers._isCarParking = function(chunk, context, bodies, params) {
+    // strip the whitespace and lowercase content.name just to be safe and return true or false appropriately
+    return params.upgradeName.toLowerCase().replace(/ /g, "") === 'carparking' ? true : false;
```

Comment:
Should there be a check for `param.upgradeName`? If no value has been passed this could break.